Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30876: Relative import from unloaded package now reimports the package
```

Comment:
FYI I installed blurb from source:

* clone /home/haypo/prog/python/core-workflow
* enter blurb/ directory
* install flit: sudo python3 -m pip install -U flit
* install blurb using flit: sudo python3 -m flit install

(I would also prefer to get it using pip ;-))